rss_pipeline/extracting_article_from_url.py

```diff
@@ -0,0 +1,47 @@
+import requests
+from bs4 import BeautifulSoup
+import requests
```

Comment:
`requests` is imported twice. As it is a built-in library, remove the second one